cmake/SSGCommon.cmake

```diff
@@ -817,12 +877,28 @@ macro(ssg_build_derivative_product ORIGINAL SHORTNAME DERIVATIVE)
     # and not configure time.
     install(
        CODE "
-       file(GLOB GUIDE_FILES \"${CMAKE_BINARY_DIR}/ssg-${DERIVATIVE}-guide-*.html\") \n
+       file(GLOB GUIDE_FILES \"${CMAKE_BINARY_DIR}/guides/ssg-${DERIVATIVE}-guide-*.html\") \n
            file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${SSG_GUIDE_INSTALL_DIR}\"
            TYPE FILE FILES \${GUIDE_FILES}
        )"
        COMPONENT doc
     )
+    install(
+       CODE "
+       file(GLOB ROLE_FILES \"${CMAKE_BINARY_DIR}/roles/ssg-${DERIVATIVE}-role-*.yml\") \n
+           file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${SSG_ROLE_INSTALL_DIR}\"
+               TYPE FILE FILES \${ROLE_FILES}
+       )"
+       COMPONENT doc
+    )
+    install(
+       CODE "
+       file(GLOB ROLE_FILES \"${CMAKE_BINARY_DIR}/roles/ssg-${DERIVATIVE}-role-*.sh\") \n
+           file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${SSG_ROLE_INSTALL_DIR}\"
+               TYPE FILE FILES \${ROLE_FILES}
+       )"
+       COMPONENT doc
```

Comment:
Crap, didn't notice these, same code in `ssg_build_derivative_product` as in ssg_`build_product`.